PolyEngine/Engine/Src/DebugWorldComponent.hpp

```diff
@@ -17,4 +17,5 @@ namespace Poly
 	private:
 		FPSSystem::FPSData FPSData;
 	};
+	COMPONENT_INS(WorldComponentsFamily, DebugWorldComponent)
```

Comment:
Let's rename this macro to REGISTER_COMPONENT.

Reply:
done